.github/workflows/ci.yml

```diff
@@ -44,9 +44,9 @@ jobs:
 
       - name: Build and Run node
         run: |
-            cargo b
-            cargo r -- --dev --tmp &
-            sleep 60
+          cargo build
```

Comment:
`cargo build --release` will be faster for the compile process.  The default build directory is `target/debug`

Reply:
`--release` will set the opt level to 3

much slower